src/hotspot/share/oops/method.hpp

```diff
@@ -436,11 +451,13 @@ class Method : public Metadata {
   int interpreter_invocation_count()            { return invocation_count();          }
 
 #ifndef PRODUCT
-  int  compiled_invocation_count() const        { return _compiled_invocation_count;  }
-  void set_compiled_invocation_count(int count) { _compiled_invocation_count = count; }
+  int      compiled_invocation_count() const    { return _compiled_invocation_count;  }
+  int64_t  compiled_invocation_count64() const  { return _compiled_invocation_count64;}
+  void set_compiled_invocation_count(int count) { _compiled_invocation_count64 = (int64_t)count; }
 #else
   // for PrintMethodData in a product build
-  int  compiled_invocation_count() const        { return 0;  }
+  int      compiled_invocation_count() const    { return 0; }
```

Comment:
`compiled_invocation_count()` looks unused.

Reply:
compiled_invocation_count() was used in compare_methods() and collect_invoked_methods(). I have converted those call sites to compiled_invocation_count64().